github-users-web-client/src/main/java/com/xemantic/githubusers/web/GitHubUsersModule.java

```diff
@@ -71,11 +75,20 @@ static String getProjectGitHubUrl() {
     return "https://github.com/xemantic/github-users-web";
   }
 
-  @Provides
-  @Singleton
-  static EventBus getEventBus() {
-    return new DefaultEventBus();
-  }
+  // Snackbar message event bus
```

Comment:
Eventually, this can be auto-generated using APT, so the generator can find all Events in the project and generate an EventBusModule with all Subjects and bindings.